cgi-bin/dashboard-aii.cgi

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use Config::Simple;
+use JSON::XS;
+use Switch;
+use CGI;
+
+# Configuration file to boot from HD. Following pxelinux convention
+# it must be called default so if the <IpAddressInHex> link
+# is missing the node will boot from the hard disk
+my $boothd = "default";
+
+# Quattor profiles
+my $web_root = "/var/www/html";
+my $profiles_path = "profiles";
+my $profiles_dir = $web_root.'/'.$profiles_path;
+
+# Read AII configuration
+$cfg = new Config::Simple();
```

Comment:
maybe use `Config::Tiny` if possible, since that is standard part of quattor (but i'm not sure it can handle the format)